ui/packages/tidb-dashboard-lib/src/apps/SlowQuery/translations/en.yaml

```diff
@@ -109,6 +109,10 @@ slow_query:
     status:
       success: Success
       error: Failed
+  overview:
+    empty_result: Great, the cluster has no any related slow queries!
```

Comment:
Why do we congratulate the user when they did not get the result they want? How about showing empty result tips only?

Reply:
Is it not a good news if the cluster has no slow queries? ^_^